psql2mysql/__init__.py

```diff
@@ -194,16 +199,22 @@ class TargetDatabaseEmpty(Exception):
 
 
 class DbDataMigrator(object):
-    def __init__(self, config, source, target):
+    def __init__(self, config, source, target, chunk_size=None):
```

Comment:
Actually, we're going about this all wrong, just set the default to cfg.CONF.chunk_size. oslo.config takes care of it being a number for us.